pkgs/development/tools/protoc-gen-go-vtproto/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+let
+  version = "0.2.0";
+in
+buildGoModule {
+  inherit version;
+
+  pname = "protoc-gen-go-vtproto";
+
+  src = fetchFromGitHub {
+    owner = "planetscale";
+    repo = "vtprotobuf";
+    rev = "v${version}";
+    sha256 = "0kjjpfsiws4vi36ha1gajb97rwcggqw753mv2jqf09kdfszz9p63";
+  };
+
+  vendorSha256 = "01lxwlgh3y3gp22gk5qx7r60c1j63pnpi6jnri8gf2lmiiib8fdc";
+
+  # conformance_test.go:55: execution error: fork/exec conformance/conformance-test-runner: no such file or directory
```

Comment:
Do we need to create $HOME or similar?

Reply:
No, it's trying to run the `conformance-test-runner` from the protobuf source tree. This test isn't normally run unless you're developing vtproto itself. I can turn it off with `excludedPackages = "\\conformance";`, if that's better?